workshops/forms.py

```diff
@@ -88,17 +88,8 @@ class BootstrapHelperFormsetInline(BootstrapHelper):
 bootstrap_helper_inline_formsets = BootstrapHelperFormsetInline()
 
 
-class InstructorsForm(forms.Form):
+class WorkshopStaffForm(forms.Form):
     '''Represent instructor matching form.'''
-
-    latitude = forms.FloatField(label='Latitude',
-                                min_value=-90.0,
-                                max_value=90.0,
-                                required=False)
-    longitude = forms.FloatField(label='Longitude',
-                                 min_value=-180.0,
-                                 max_value=180.0,
-                                 required=False)
```

Comment:
Is that really that popular scenario?

@maneesha any strong feelings about leaving/removing lat/long search from "Find instructors" form (now "Find Workshop Staff" form)?